admin-dev/themes/new-theme/js/components/text-to-link-rewrite-copier.js

```diff
@@ -65,7 +65,9 @@ const textToLinkRewriteCopier = (
   },
 ) => {
   $(document).on(options.eventName, `${sourceElementSelector}`, (event) => {
-    $(destinationElementSelector).val(window.str2url($(event.currentTarget).val(), 'UTF-8'));
+    if (window.location.href.indexOf('new') > -1) {
```

Comment:
I understand the purpose, however it means you can never benefit of this feature when you are editing the content and you do want to update it
I would propose two solutions for this:
- ideally, we should have a button next to the rewrite-link input to force its update manually (that's the long solution)
- you could check in your condition if the input is empty, so there would be a workaround if you need to refresh it you can empty the input 🤔

Reply:
1- there is no benefit on adding a redirection, if the merchant still wants a rename, it can change it manually.
2- the rewrite can't be empty at creation time : it's mandatory

Reply:
@jolelievre @ldeprestashop in product page, the button in your first case has been deleted (experimental version)
The "Force update of friendly URL" option in Product Settings is the best way IMO, so can we consider that this settings should also be relevant in category page ?

Reply:
I'm afraid checking for `new` in URL is really not the right way to do it.
If you have `www.newstyle.fr` it will return true every time.

Reply:
@PierreRambaud what do you suggest ? There is no hidden id in the page.

Reply:
You can retrieve an element or create a variable to pass into javascript instead of checking the URL.

Reply:
@PierreRambaud ok for that.
@PierreRambaud @ldeprestashop What do you think of using "Force update of friendly URL" option in Product Settings page ?